docs/environment.yaml

```diff
@@ -0,0 +1,22 @@
+name: bw_timex_documentation
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  # core functionality
+  - python=3.11
+  - ipython
+  # sphinx
+  - sphinx=7.3.7 # core builder # https://anaconda.org/conda-forge/sphinx/files
+  # theme and extensions
+  - pydata-sphinx-theme=0.15.3 # website theme # https://anaconda.org/conda-forge/pydata-sphinx-theme/files
```

Comment:
@jakobsarthur, these are the pinned versions I was referring to. You can always bump the package versions when either:

1. You have tested the build with the new versions locally
2. the docs.brightway.dev repo has bumped them